libs/@blockprotocol/core/src/types.ts

```diff
@@ -101,9 +99,9 @@ export type BlockMetadata = {
    */
   repository?: BlockMetadataRepository | null;
   /**
-   * The versioned URL to the block's schema
+   * The versioned URL to the block's schema, e.g. https://blockprotocol.org/@blockprotocol/types/entity-type/thing/v/2
    */
-  schema: VersionedUrl;
+  schema: string;
```

Comment:
:(

Reply:
I know :(

Reply:
We can patch the generated JSON schema to bring it back